### PR TITLE
open api no longer require the active part of the url

### PR DESCRIPTION
**Pull request by lukeapage**

### CODE REVIEW

File: test/unit/openapi/streaming/subscription-spec.js [hunk 1]
```diff
@@ -37,14 +37,14 @@ describe("openapi StreamingSubscription", () => {
...
-            expect(transport.post.calls.argsFor(0)).toEqual(['serviceGroup', 'test/resource/active', null, jasmine.objectContaining({ body: jasmine.objectContaining({ RefreshRate: 120 }) })]);
+            expect(transport.post.calls.argsFor(0)).toEqual(['serviceGroup', 'test/resource', null, jasmine.objectContaining({ body: jasmine.objectContaining({ RefreshRate: 120 }) })]);
         });
         it("has a minimum refresh rate", () => {
             var subscription = new Subscription('123', transport, 'serviceGroup', 'test/resource', { RefreshRate: 1 });
             subscription.onSubscribe();
 
             expect(transport.post.calls.count()).toEqual(1);
-            expect(transport.post.calls.argsFor(0)).toEqual(['serviceGroup', 'test/resource/active', null, jasmine.objectContaining({ body: jasmine.objectContaining({ RefreshRate: 100 }) })]);
+            expect(transport.post.calls.argsFor(0)).toEqual(['serviceGroup', 'test/resource', null, jasmine.objectContaining({ body: jasmine.objectContaining({ RefreshRate: 100 }) })]);
```
Comment (michax) [inline test/unit/openapi/streaming/subscription-spec.js]: Nitpick. We are testing transport.post for subscribe. However we don't validate it for patches. Maybe it would be worth adding same unit test, that will assert built url for patch. However, it could be task for another PR :)

Comment (lukeapage) [inline test/unit/openapi/streaming/subscription-spec.js]: its a good point..

Comment (lukeapage) [inline test/unit/openapi/streaming/subscription-spec.js]: Done
